ignite/engine/engine.py

```diff
@@ -118,6 +118,8 @@ def compute_mean_std(engine, batch):
 
     """
 
+    state: State
```

Comment:
Please, explain why to add these lines ?

Reply:
It is for adding type hints in attributes, but this should be done with upgrade sphinx 3.4 PR.
Removing now...